modules/components/pages/outputs/snowflake_put.adoc

```diff
@@ -9,6 +9,7 @@
 
 component_type_dropdown::[]
 
+NOTE: For improved performance, cost effectiveness and ease of use, Redpanda recommends that you migrate to the xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output].
```

Comment:
```suggestion
TIP: Use the xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output] for improved performance, cost-effectiveness, and ease of use.
```

We don't have a migration guide and migrate only applies to existing users of this connector. For new users, we are simply recommending that they don't use this one and instead use the `snowflake_streaming` output.

Reply:
Updated